spec/models/user_spec.rb

```diff
@@ -9,9 +9,28 @@
   end
 
   describe "validations" do
-    it { is_expected.to validate_presence_of(:first_name) }
-    it { is_expected.to validate_presence_of(:last_name) }
-    it { is_expected.to validate_presence_of(:email) }
+    subject { FactoryBot.create(:user) }
```

Comment:
I doubt this line is necessary

Reply:
We need this because of the `validate_uniqueness_of`. Read this for details: https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_record/validate_uniqueness_of_matcher.rb#L56

Reply:
Please correct me if there's a better way!

Reply:
👍 Looks like that's the only way